GM-SC-CHARTER.adoc

```diff
@@ -1,15 +1,20 @@
+= Charter of The TODO Group General Members (GM) and Steering Committee (SC)
+
```

Comment:
```suggestion
*Disclaimer: Please note that this charter only covers the rights, criteria, and responsibilities of general members and the steering committee. It does not address other actors within the TODO Group, such as OSPO associate organizations and the individual community of contributors and OSPO practitioners. For information regarding these additional groups, please consult the [TODO Group community & network page](https://todogroup.org/community/)*
```

Reply:
I think I know where you are going with this and why, but this disclaimer might be misleading rather than clarifying. For instance, it is not entirely true and accurate: The charter also talks about IP Policy (section 10), which applies to all participants, not just members. The same should be true for some of the operating principles, e.g., meetings are under Chatham House rules - for all I hope, not just members.

So, lacking a better suggestion right now, I'd rather opt not to include this.

Reply:
Perhaps this is a better wording:

Please note this charter covers the activities and responsibilities of general members and the steering committee. There are many other folks active in TODO Group, such as individuals employed by OSPO Associate Organizations and individuals who are OSPO practitioners and contributors to open source projects and communities.  Individuals persons are welcome and encouraged to join and participate in the TODO Group as long as they abide by our code of conduct. More information on the [OSPO Associates](https://todogroup.org/associates/) page.

--- 
Side note, I am delighted we've arrived at this wording and made it abundantly clear that you do not have to be employed to participate in TODO. That was a big oversight and I am grateful to @annania for bringing it to our attention. Making this change is particularly timely given current market conditions and their impact to OSPO professionals.